FirebaseRTC/public/index.html

```diff
@@ -22,10 +22,10 @@
 
      <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="main.css">
-     <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+     <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
```

Comment:
Does this work? Should the href be relative to this file...so:

href="../assets/css/bootstram.min.css"

The _assets_ folder is up one levely thus the "../" -- is that correct? @lillianchou94